roles/cfg_openwrt/templates/corerouter/bird.conf.j2

```diff
@@ -0,0 +1,104 @@
+#jinja2: trim_blocks: "true", lstrip_blocks: "true"
+{% import 'libraries/network.j2' as libnetwork with context %}
+
+log syslog all;
+debug protocols all;
+
+#master4 is implicitly created
+ipv6 sadr table v6_main;
+ipv4 table v4_babel_ff;
+ipv4 table v4_babel_default;
+
+protocol device {
+};
+
+protocol direct {
+	interface "lo";
+	ipv6 sadr {
+		table v6_main;
+	};
+}
+
+protocol babel {
+	randomize router id yes;
+	ipv6 sadr {
+		table v6_main;
+		# Import only /56 location aggregates and default routes
+		import where (net.len = 56) || (net = ::/0 from {{ freifunk_global_prefix }} );
+		export filter {
+			if net = {{ ipv6_prefix }} from ::/0 then accept;
+			if source = RTS_BABEL then accept;
+			reject;
+		};
+	};
+	ipv4 {
+		import all;
+		export where source = RTS_BABEL; # For now only advertise routes learned from babel
+	};
+
+	# Mesh interfaces
+{% for nw in networks | selectattr('role', 'equalto', 'mesh') %}
+  {% set default_mesh_metric = mesh_metric_adhoc_in if 'mesh_ap' else mesh_metric_default_in %}
+	interface "{{ libnetwork.getIfname(nw) }}" {
+		type wireless;
+		rxcost {{ nw.get('mesh_metric', default_mesh_metric) }};
+	};
+{% endfor %}
+	# Tunnel interfaces provided by tunspace
+{% for nw in networks | selectattr('role', 'equalto', 'tunnel') %}
+	interface "{{ nw.get('ifname') }}" {
+		type wireless;
```

Comment:
Should be `tunnel` instead of `wireless` to speed up the process of dropping routes e.g. if one of multiple tunnel servers becomes unreachable.

Reply:
https://bird.network.cz/?get_doc&v=20&f=bird-6.html#ss6.2
> 
> type wired|wireless|tunnel
> 
> >     This option specifies the interface type: Wired, wireless or tunnel. On wired interfaces a neighbor is considered unreachable after a small number of Hello packets are lost, as described by limit option. On wireless interfaces the ETX link quality estimation technique is used to compute the metrics of routes discovered over this interface. This technique will gradually degrade the metric of routes when packets are lost rather than the more binary up/down mechanism of wired type links. A tunnel is like a wired interface, but turns on RTT-based metrics with a default cost of 96. Default: wired. 


While i think this is definately considerable, at least to change that to wired, i would like to keep it out. Tbh, changing any fundamential routing parameters without good testing in the field is out of scope for this PR.